agent/containermetadata/parse_metadata_test.go

```diff
@@ -218,7 +218,8 @@ func TestParseHasNetworkSettingsNetworksEmpty(t *testing.T) {
 	mockHostConfig := &dockercontainer.HostConfig{NetworkMode: "bridge"}
 	mockNetworkSettings := &types.NetworkSettings{
 		DefaultNetworkSettings: types.DefaultNetworkSettings{
-			IPAddress: "0.0.0.0",
+			IPAddress:         "1.2.3.4",
+			GlobalIPv6Address: "5:6:7:8::",
```

Comment:
Do we have another test case where `GlobalIPv6Address` does not exist? We need that because `GlobalIPv6Address` might not always exist.